server.py

```diff
@@ -66,7 +66,7 @@ def proxy_site(env, start_response, video=False):
         # such as 8192 lest that causes the socket library to limit the
         # TCP window size
         # Might need fine-tuning, since this gives us 4*65536
-        # The tradeoff is that larger values (such as 6 seconds) only 
+        # The tradeoff is that larger values (such as 6 seconds) only
```

Comment:
Avoid these kinds of unrelated changes in the future. They introduce noise in the pull request and mess up git blame. Code cleaning, removing extraneous spaces, and PEP compliance can be done all at once at a later time.

Reply:
#34